avocado/core/test.py

```diff
@@ -978,6 +1010,52 @@ class ReplaySkipTest(SkipTest):
     _skip_reason = "Test skipped due to a job replay filter!"
 
 
+class ReplayResultCopier(Test):
```

Comment:
This is another point of discussion. As a concept, a skipped test is a test that is not executed. We cannot say we skipped the test and still it contains the execution information. If that' s a requirement for you, than we should introduce a new status, like ` RECOVERED`, with proper specification and documentation. But in my view, I'd just skip it and don't populate it's results.